AEPIntegrationTests/IdentityIntegrationTests.swift

```diff
@@ -204,4 +204,43 @@ class IdentityIntegrationTests: XCTestCase {
         wait(for: [requestExpectation], timeout: 2)
     }
 
+    /// Tests that when we reset the identities we generate a new ECID and send it out
+    func testResetIdentities() {
+        // set first ecid
+        var props = IdentityProperties()
+        let firstEcid = ECID()
+        props.ecid = firstEcid
+        props.saveToPersistence()
+
+        initExtensionsAndWait()
+
+        let requestExpectation = XCTestExpectation(description: "identity hits go out")
+        requestExpectation.expectedFulfillmentCount = 2
```

Comment:
for the test expectation this should be 1, right? let's wait and clear any other network requests sent out before `MobileCore.resetIdentities()`